common/math-transforms.js

```diff
@@ -22,16 +22,20 @@
            Since these are in tree-order,
            if we process them
            in reverse order we should side-step
-           the gnarliest of potentialnesting
+           the gnarliest of potential nesting
            issues, I think
         */
           matches.reverse().forEach(el => {
-            el.setAttribute("data-math-transformed", "");
-            let copy = el.cloneNode(true);
-            let transformed = transformer(copy);
-  
-            recurse = true;
-            el.replaceWith(transformed || copy);
+            const nextChild = el.nextSibling;
+            const parent = el.parentElement;
+            let transformed = transformer(el);
```

Comment:
just out of curiosity, is there a reason this one is let, but the others are const? I don't see it changing somewhere?